backend/src/packages/chaiNNer_standard/image_utility/miscellaneous/generate_hash.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+import base64
+import hashlib
+from typing import Tuple
+
+import numpy as np
+
+from nodes.impl.image_utils import to_uint8
+from nodes.properties.inputs import ImageInput, SliderInput
+from nodes.properties.outputs import TextOutput
+
+from .. import miscellaneous_group
+
+
+@miscellaneous_group.register(
+    schema_id="chainner:image:generate_hash",
+    name="Generate Hash",
+    description="Generate a hash from an image using the BLAKE2b hashing algorithm.",
+    icon="MdCalculate",
+    inputs=[
+        ImageInput(),
+        SliderInput(
+            "Digest Size (in bytes)",
+            minimum=1,
+            maximum=64,
+            default=8,
+            precision=1,
+            controls_step=1,
+        ),
+    ],
+    outputs=[
+        TextOutput("Hex", output_type="string"),
+        TextOutput("Base64", output_type="string"),

```

Comment:
`string` is the default, so no need to specify it.

```suggestion
        TextOutput("Hex"),
        TextOutput("Base64"),
```